src/binding/c/timer_api.txt

```diff
@@ -0,0 +1,60 @@
+# vim: set ft=c:
+
+MPI_Wtick:
+    .desc: Returns the resolution of MPI_Wtime
+    .impl: direct
+    .skip: ThreadSafe, Fortran
+    .seealso: MPI_Wtick, MPI_Comm_get_attr, MPI_Attr_get
+/*
+  Return value:
+  Time in seconds since an arbitrary time in the past.
+
+  Notes:
+  This is intended to be a high-resolution, elapsed (or wall) clock.
+  See 'MPI_WTICK' to determine the resolution of 'MPI_WTIME'.
+  If the attribute 'MPI_WTIME_IS_GLOBAL' is defined and true, then the
+  value is synchronized across all processes in 'MPI_COMM_WORLD'.
+
+  Notes for Fortran:
+  This is a function, declared as 'DOUBLE PRECISION MPI_WTIME()' in Fortran.
```

Comment:
Can we omit the following since this routine does not return an error?
```
.N Errors
.N MPI_SUCCESS
```

Reply:
done

Reply:
Updated python script to add .skip: Errors directive.